src/gateways/help-request.js

```diff
@@ -38,16 +38,37 @@ const ToHelpRequest = (hr) => {
         urgentEssentialsAnythingElse: hr.UrgentEssentialsAnythingElse,
         whenIsMedicinesDelivered: hr.WhenIsMedicinesDelivered,
         rescheduledAt: hr.RescheduledAt,
-        requestedDate: hr.RequestedDate
+        requestedDate: hr.RequestedDate,
+        helpRequestCalls: ToCalls(hr.HelpRequestCalls),
     };
 };
 
+const ToCalls = (calls) => {
+    return calls?.map((call) => {
+        return {
+            id: call.Id,
+            helpRequestId: call.HelpRequestId,
+            callType: call.CallType,
+            callDirection: call.CallDirection,
+            callOutcome: call.CallOutcome,
+            callDateTime: call.CallDateTime
+        };
+    });
+};
+
 export class HelpRequestGateway extends DefaultGateway {
     async getHelpRequest(residentId, requestId) {
         const response = await this.getFromUrl(`resident/${residentId}/helpRequests/${requestId}`);
         const helpRequest = ToHelpRequest(response);
         return helpRequest;
     }
+
+    async getHelpRequests(residentId) {
+        const response = await this.getFromUrl(`resident/${residentId}/helpRequests`);
+        const helpRequests = response.map((x) => ToHelpRequest(x));
```

Comment:
Fun fact, you can do this 

```suggestion
        const helpRequests = response.map(ToHelpRequest);
```

Reply:
:flips-a-table:

Reply:
Of course, it's a method 🤦